sdk/security/keyvault/azadmin/build.go

```diff
@@ -1,12 +1,11 @@
-//go:build go1.18
-// +build go1.18
-
-//go:generate autorest ./settings/autorest.md
-//go:generate autorest ./rbac/autorest.md
-//go:generate autorest ./backup/autorest.md
+//go:generate tsp-client update --output-dir ./settings
+//go:generate tsp-client update --output-dir ./rbac
+//go:generate tsp-client update --output-dir ./backup
+//go:generate go run ./internal/generate/transforms.go
 //go:generate gofmt -w .
+//go:generate rm ./backup/constants.go
 
 // Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
+// Licensed und er the MIT License. See License.txt in the project root for license information.
```

Comment:
👀

Reply:
Thanks! Fixed in [1d65168](https://github.com/Azure/azure-sdk-for-go/pull/23906/commits/1d651689c7aceb11db84aa7fb98d9123b9e662c1)